src/Pages/ErrorPages/Error.cshtml.cs

```diff
@@ -0,0 +1,31 @@
+using System.Diagnostics;
+using System.IO;
+using Microsoft.AspNetCore.Diagnostics;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.RazorPages;
+using Microsoft.Extensions.Logging;
+
+namespace ePiggyWeb.Pages.ErrorPages
+{
+    [ResponseCache(Duration = 0, Location = ResponseCacheLocation.None, NoStore = true)]
```

Comment:
**Response caching reduces the number of requests a client or proxy makes to a web server. Response caching also reduces the amount of work the web server performs to generate a response. Response caching is controlled by headers that specify how you want client, proxy, and middleware to cache responses.** 
- The ResponseCacheAttribute specifies the parameters necessary for setting appropriate headers in response caching.
 - To enable caching, Duration must be set to a positive value 
- NoStore overrides most of the other properties. When this property is set to true, the Cache-Control header is set to no-store. NoStore is typically set to true for error pages. 
 - Setting Location to None sets both Cache-Control and Pragma headers to no-cache.
I'm myself do not understand this completely but it just sets everything up so it would be handled like error page.
[Source](https://docs.microsoft.com/en-us/aspnet/core/performance/caching/response?view=aspnetcore-5.0)